engine/src/main/java/org/pentaho/di/job/entries/ping/JobEntryPing.java

```diff
@@ -245,6 +245,11 @@ public String getRealHostname() {
   }
 
   public String getTimeOut() {
```

Comment:
Since `getTimeOut()` is not a simple getter method. You need to add a javadoc comment explaining the function. Specifically what happens with `timeout` is not numeric.

Reply:
Should I just comment this -
"Incase, if a kjb/ktr already has non-numeric value, 0 & negative integers ,value will be replaced with 3000 ( default value) during runtime and user will be notified about this replacement in logs"  ?